app/views/paye/HistoricPayAsYouEarnView.scala.html

```diff
@@ -47,23 +47,31 @@
     ptaHeader(
         mainHeadingText = messages("tai.paye.heading", TaxPeriodLabelService.taxPeriodLabel(viewModel.taxYear.year)),
         preHeadingText = request.fullName,
-        displayBackLink = true,
-        customBackLinkContent = Some(backLinkToChooseTaxYear))
+        displayBackLink = false,
+        customBackLinkContent = Some(backLinkToChooseTaxYear),
+        showGdsVersion = true)
 }
 
-@mainTemplate(
+@main(
     title = messages("tai.paye.heading", TaxPeriodLabelService.taxPeriodLabel(viewModel.taxYear.year)),
-    articleClasses = Some("full-width"),
     authedUser = Some(request.taiUser),
-    pageHeader = Some(header),
+    pageTitle = Some("Check your Income Tax"),
```

Comment:
shouldn't this come from messages?

Reply:
[](url
<img width="1236" alt="Screen Shot 2022-04-21 at 15 50 23" src="https://user-images.githubusercontent.com/78593709/164485062-28e9acda-089f-40db-aa96-55152dcf00f0.png">
)